src/constants/permissions.ts

```diff
@@ -239,9 +275,6 @@ values.SYSCONSOLE_ANCILLARY_PERMISSIONS = {
     [values.SYSCONSOLE_WRITE_ENVIRONMENT]: [
         values.MANAGE_JOBS,
     ],
-    [values.SYSCONSOLE_WRITE_SITE]: [
```

Comment:
Also , you’ll need to merge the latest changes from SectionsToSubSections branches into your personal branch. The new changes are getting rid of all the ancillary permissions and moving that logic from the front end to the back end. You won’t need to manage those yourself anymore.